doc/v1/docs/api_subs.md

```diff
@@ -234,7 +234,7 @@ Please refer to section [Errors](api_errors.md) to see all possible Errors
 
 
 ## [POST] Sending an ACK
-This request sends an ack that the message has been  received
+This request sends an ack that a message has been received. The request accepts a list of ackIDs. The ackID with the maximum offset will be acknowledged
```

Comment:
How about:  "Messages retrieved from a pull subscription can be acknowledged by sending message with an array of ackIDs. In the current implementation, the service will retrieve the ackID corresponding to the highest message offset and will consider that message and all previous messages as acknowledged by the consumer." ?